File: C1.UWP.ExpressionEditor/CS/ExpressionEditorSamples/Data/PageCache.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ExpressionEditorSamples
{
    public static class PageCache
    {
        // Expression Cache
        static string _expressionCache = "";
        static string _fieldCache = "";

        static public string GetCacheExpression()
        {
            return _expressionCache;
        }

        static public void SetCacheExpression(string expression)
        {
            _expressionCache = expression;
        }

        static public string GetCacheField()
        {
            return _fieldCache;
        }

        static public void SetCacheField(string field)
        {
            _fieldCache = field;
        }

    }
}

File: C1.UWP.ExpressionEditor/VB/ExpressionEditorSamples/Data/PageCache.vb
Public Class PageCache
    ' Expression Cache
    Shared _expressionCache As String = ""
    Shared _fieldCache As String = ""

    Public Shared Function GetCacheExpression() As String
        Return _expressionCache
    End Function

    Public Shared Sub SetCacheExpression(expression As String)
        _expressionCache = expression
    End Sub

    Public Shared Function GetCacheField() As String
        Return _fieldCache
    End Function

    Public Shared Sub SetCacheField(field As String)
        _fieldCache = field
    End Sub
End Class
